tools/ci/tc/tasks/test.yml

```diff
@@ -296,9 +296,13 @@ tasks:
         - vars:
             browser: firefox
             channel: nightly
+            exclude-users:
```

Comment:
I wonder if we should call this variable `stability-exclude-users` or something, since it's only used for the stablity jobs. The actual key in the `do` section can still be `exclude-users` of course.

Reply:
Good clarity change - added!